demisto_sdk/commands/common/git_util.py

```diff
@@ -795,3 +795,12 @@ def get_all_changed_files(
             get_only_current_file_names=True,
         )
         return modified_files.union(added_files).union(renamed_files)
+
+    def _is_file_ignored(self, file_path: str) -> bool:
```

Comment:
suggestion: change to `_is_file_git_ignored`, to distinguish from the method that checks whether the file name is in the constant of ignored files.